fog/ingest/server/src/worker.rs

```diff
@@ -254,14 +255,26 @@ impl PeerCheckupWorker {
         let stop_requested = Arc::new(AtomicBool::new(false));
         Self {
             stop_requested: stop_requested.clone(),
-            thread: Some(std::thread::spawn(move || loop {
-                if stop_requested.load(Ordering::SeqCst) {
-                    log::info!(logger, "Stop Requested: Polling loop stopped",);
-                    break;
-                }
+            thread: Some(std::thread::spawn(move || {
+                log::debug!(logger, "Peer checkup thread started");
+
+                let mut last_refreshed_at = Instant::now();
+
+                loop {
+                    if stop_requested.load(Ordering::SeqCst) {
+                        log::debug!(logger, "Peer checkup thread stop requested.");
+                        break;
+                    }
 
-                controller.peer_checkup();
-                std::thread::sleep(peer_checkup_period);
```

Comment:
this change was tangential to the goal of the PR, but i realized that this thread is sleeping for like 60 seconds. that probably makes the tests take at least 60 seconds. so if we make this only sleep 1 second at a time, same as consensus thread, this should make the unit tests run a lot faster.

Reply:
Good find